### ISSUE TITLE
Training speed and CD for the ShapeNet validation set

### ISSUE THREAD

**Issue by Jiayuan-Gu** [TonythePlaneswalker/pcn#32]
Thanks for the good work.
However, since the test data are removed due to completion3d, can you report CD on the ShapeNet validation set? Moreover, it seems to be relatively slow to train, about 1.2s/batch on one GTX2080Ti. Can you also report the training speed or training time for ShapeNet?

**Comment by TonythePlaneswalker** [comment]
The CD for the validation set is similar to the ones on the test set. You should be able to get these numbers by running the pre-trained model on the validation set. The training takes around 36 hours in total, i.e. about 0.45s per iteration on an 1080 Ti.

**Comment by Skye2099** [comment]
> Thanks for the good work. However, since the test data are removed due to completion3d, can you report CD on the ShapeNet validation set? Moreover, it seems to be relatively slow to train, about 1.2s/batch on one GTX2080Ti. Can you also report the training speed or training time for ShapeNet?

hi, did you use the tensorflow-gpu==1.12.0 or tensorflow==1.12.0 ？ i dont konw why my train process not use gpu,thank you so much
